src/TitleBindPieces/TitleBindPiecesContainer/TitleBindPiecesContainer.js

```diff
@@ -146,6 +155,7 @@ export const TitleBindPiecesContainer = () => {
         onConfirm={onConfirm}
         showDeleteMessage={showDeleteMessage}
```

Comment:
it sounds like a function, but in fact it's boolean

Reply:
additionally why do you call it delete when it's responsible for transfer actions? 

additionally I think you just need to pass openRequests to the component instead of this flag, barcodes etc and component will decide what to do with this data

Reply:
There is one more scenario(5) to delete requests for ECS env from the task https://folio-org.atlassian.net/browse/UIREC-353